content/en/project/status/_index.md

```diff
@@ -7,18 +7,26 @@ weight: 50
 ## Annual Reports
 
 The Medley team has produced annual reports documenting their major achievements.
+Sneak Peek: We are working on our [2023 Medley Interlisp Annual Report](https://docs.google.com/document/d/1yL1WeYvgu5JstWyjD3fWl1mDZv8kIr_U3_1onSWvUko/edit?usp=sharing). Still a work in progress -- don't distribute until we announce it ready.
+## 12/2022
+* [2022 Annual Report](/project/status/2022medleyannualreport)
 
-- [2021 Annual Report](/project/status/2021medleyannualreport)
-- [2022 Annual Report](/project/status/2022medleyannualreport)
+## Talks
 
-## March 2023 Bay Area Lisp Meeting
+### CCS Talk
 
-An overview of the Medley Interlisp project was presented at the March Bay Area
-Lisp Meeting.  The following recording provides an introduction to the work
-being undertaken by the Medley project.
+
+### 2023 Bay Area Lisp Meeting
+
+An overview of the Medley Interlisp project was presented at the March 2023 Bay Area Lisp Meeting.  The following recording provides an introduction to the work
+being undertaken by the Medley Interlisp Project.
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/N1MobfEaoWY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
-## Closed Issues
+Drafts and final versions of slide decks can be found [here](https://drive.google.com/drive/folders/1wQZ9F-doxx5gsGaQ7gKtOa5AObJalNJ9?usp=sharing)
+
+
+### Closed Issues
 
 The [issues closed](https://github.com/Interlisp/medley/issues?q=is%3Aissue+is%3Aclosed) by the team can be found in our GitHub repository.
+<!-- is there a way to ask for "closed in the last N days"?  -->
```

Comment:
Issues closed in 2022:  [https://github.com/Interlisp/medley/issues?page=6&q=is%3Aissue+closed%3A%3E2022-12-31](https://github.com/Interlisp/medley/issues?page=6&q=is%3Aissue+closed%3A%3E2022-12-31)

I don't see a way of getting the last n days from the day the query is run.